src/validation.h

```diff
@@ -303,6 +303,8 @@ class CScriptCheck
     ScriptError GetScriptError() const { return error; }
 };
 
+static_assert(std::is_move_constructible_v<CScriptCheck>);
```

Comment:
78030e0c08ce8ab30bde05ce97def571f05f1399: Not sure what the point of this is, given that it will be even true if there is a *copy* constructor.

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/26749#issuecomment-1370149202).